cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaQuorumCheck.java

```diff
@@ -89,7 +90,8 @@ private boolean isQuorumHealthyWithoutPod(int podId, QuorumInfo info) {
                     LOGGER.debugCr(reconciliation, "The controller {} has caught up with the quorum leader", replicaId);
                     LOGGER.debugCr(reconciliation, "The lastCaughtUpTimestamp for controller {} is {}", replicaId, lastCaughtUpTimestamp);
                 } else {
-                    LOGGER.debugCr(reconciliation, "The controller {} is behind from the leader", replicaId);
+                    LOGGER.debugCr(reconciliation, "The controller {} has fallen behind the leader", replicaId);
+                    LOGGER.debugCr(reconciliation, "The lastCaughtUpTimestamp for controller {} is {}", replicaId, lastCaughtUpTimestamp);
```

Comment:
Instead of duplicating the line, I think we can put them at L85, before if/else block, maybe?

Reply:
That makes more sense, I have updated the code.